cmd/scheduled-feed/main.go

```diff
@@ -22,10 +22,17 @@ type FeedHandler struct {
 	scheduler *scheduler.Scheduler
 	pub       publisher.Publisher
 	delta     time.Duration
+	lastPoll  time.Time
 }
 
 func (handler *FeedHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
-	cutoff := time.Now().UTC().Add(-handler.delta)
+	var cutoff time.Time
+	if handler.lastPoll.IsZero() {
+		handler.lastPoll = time.Now().UTC()
```

Comment:
Will this only happen once during first poll due to `IsZero()`

Reply:
Should do, as it stands

Reply:
We want `lastPoll` to be set every time we poll though right?

Reply:
Yep.... should be rectified now